### PR TITLE
[Merged by Bors] - feature: Implement multiple partitions consumer

### PR DESCRIPTION
**Pull request by morenol**
Closes https://github.com/infinyon/fluvio/issues/1568

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -109,6 +109,16 @@ election-test: test-setup
 		${TEST_ARG_DEVELOP} \
 		${TEST_ARG_EXTRA}
 
+multiple-partition-test: TEST_ARG_EXTRA=--local $(EXTRA_ARG)
```
Comment (sehz) [inline Makefile:112]: is there reason why this only runs on local? this can be removed

**Review by sehz — CHANGES_REQUESTED** [review]
Nicely done.!   A minor nit.   Also, add to changelog

**Comment by sehz** [comment]
CI is failling.  Look at other test and make sure right args are passed

**Comment by morenol** [comment]
> CI is failling. Look at other test and make sure right args are passed

The difference that I see is that the other tests are using `--local`. I added again this line:
```
 multiple-partition-test: TEST_ARG_EXTRA=--local $(EXTRA_ARG)
```

**Review by sehz — APPROVED** [review]

**Pull request closed without merging**